### ISSUE TITLE
nrlint not working (raises no warnings/erros)

### ISSUE THREAD

**Issue by protonmw** [node-red/nrlint#43]
### Current Behavior

lint tab is available, settings has been made
none of the rule violations are reported

for more details see here: [https://discourse.nodered.org/t/nrlint-install-error/84731/9?u=protonmw](https://discourse.nodered.org/t/nrlint-install-error/84731/9?u=protonmw)

### Example flow

```
[
    {
        "id": "e6044d460bd8f44d",
        "type": "inject",
        "z": "a86e5a108d2f7b13",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "",
        "payloadType": "date",
        "x": 1000,
        "y": 1740,
        "wires": [
            [
                "52e62976f14f7ba3"
            ]
        ]
    },
    {
        "id": "52e62976f14f7ba3",
        "type": "change",
        "z": "a86e5a108d2f7b13",
        "name": "",
        "rules": [
            {
                "t": "set",
                "p": "payload",
                "pt": "msg",
                "to": "",
                "tot": "str"
            }
        ],
        "action": "",
        "property": "",
        "from": "",
        "to": "",
        "reg": false,
        "x": 1220,
        "y": 1740,
        "wires": [
            [
                "ede84a784d16fbf0"
            ]
        ]
    },
    {
        "id": "ede84a784d16fbf0",
        "type": "change",
        "z": "a86e5a108d2f7b13",
        "name": "",
        "rules": [
            {
                "t": "set",
                "p": "payload",
                "pt": "msg",
                "to": "",
                "tot": "str"
            }
        ],
        "action": "",
        "property": "",
        "from": "",
        "to": "",
        "reg": false,
        "x": 1220,
        "y": 1800,
        "wires": [
            [
                "52e62976f14f7ba3"
            ]
        ]
    }
]
```

### Environment

- nrlint version: 1.1.0
- Node-RED version: 3.1.3
- Node.js version: 18.19.0
- npm version: 10.2.3
- Platform/OS: Linux raspberrypi3Bplus 5.10.103-v7+
- Browser: Google Chrome Version 120.0.6099.200 (Offizieller Build) (64-Bit)

**Comment by protonmw** [comment]
Problem is caused by a function node with `"\r"` in code.
See here: [https://discourse.nodered.org/t/nrlint-install-error/84731/24?u=protonmw](https://discourse.nodered.org/t/nrlint-install-error/84731/24?u=protonmw)

**Comment by protonmw** [comment]
SOLVED with V1.2.0

**Comment by protonmw** [comment]
SOLVED with V1.2.0
